python/cudf/_gdf.py

```diff
@@ -131,6 +131,11 @@ def apply_mask_and(col, mask, out):
 def np_to_gdf_dtype(dtype):
     """Util to convert numpy dtype to gdf dtype.
     """
+
+    if pd.core.common.is_categorical_dtype(dtype):
```

Comment:
This is deprecated and should be removed apparently. Will use the implementation below this :).